thirdparty/glib/CMakeLists.txt

```diff
@@ -70,6 +70,8 @@ endif()
 if(DEFINED ENV{DARWIN})
     set(CFLAGS "-I/usr/local/opt/gettext/include $ENV{DARWIN_AARCH64_HOST_CFLAG} ${CFLAGS}")
     set(LDFLAGS "${LDFLAGS} -L/usr/local/opt/gettext/lib $ENV{DARWIN_AARCH64_HOST_LDFLAG} -L${LIBICONV_DIR}/lib -L${ZLIB_DIR}/lib -L${LIBFFI_DIR}/.libs -L${BINARY_DIR}/gmodule/.libs")
```

Comment:
Why is the macOS' glib build looking at `/usr/local/opt/gettext` and using the one being built?

Reply:
That's a good question ;o).

Reply:
(The mac bundle script is also bundling brew's libintl).

Don't have a device to test, so, exercise left to the apple reader ;).

Reply:
Maybe the same issue as with webp.

Reply:
Yup, I suspect chronology has a say in the matter here.

Reply:
And the macOS' CI should probably not install SDL from brew either.

Reply:
Looking a the CI build log for #1639, there are quite a few potential issues with the macOS build:
```
libtool: compile:  /usr/local/bin/ccache g++ -DHAVE_CONFIG_H -I. -I.. -O2 -DNDEBUG -I/Users/runner/work/koreader-base/koreader-base/thirdparty/leptonica/build/x86_64-apple-darwin20.6.0/leptonica-prefix/src/leptonica/src/. -I/usr/local/Cellar/cairo/1.16.0_5/include/cairo -I/usr/local/Cellar/glib/2.76.4/include -I/usr/local/Cellar/glib/2.76.4/include/glib-2.0 -I/usr/local/Cellar/glib/2.76.4/lib/glib-2.0/include -I/usr/local/opt/gettext/include -I/usr/local/Cellar/pcre2/10.42/include -I/usr/local/Cellar/pixman/0.42.2/include/pixman-1 -I/usr/local/Cellar/fontconfig/2.14.2/include -I/usr/local/opt/freetype/include/freetype2 -I/usr/local/opt/libpng/include/libpng16 -I/usr/local/Cellar/libxcb/1.15_1/include -I/usr/local/Cellar/libxrender/0.9.11/include -I/usr/local/Cellar/libxext/1.3.5/include -I/usr/local/Cellar/libx11/1.8.6/include -I/usr/local/Cellar/libxcb/1.15_1/include -I/usr/local/Cellar/libxau/1.0.11/include -I/usr/local/Cellar/libxdmcp/1.1.4/include -I/usr/local/Cellar/xorgproto/2023.2/include -I/Library/De...
```